src/App.js

```diff
@@ -1,18 +1,23 @@
-import React, { useEffect, useState, useRef } from "react";
-import { WiredButton, WiredCard, WiredInput, WiredToggle } from "wired-elements-react";
+import React, { useEffect, useState, useRef } from 'react';
+import {
+  WiredButton,
+  WiredCard,
+  WiredInput,
+  WiredToggle
+} from 'wired-elements-react';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
 import generateAcak from './libs/password-ga';
 import generateKata from './libs/password-ga-kata';
 
-import "./App.css";
+import './App.css';
 
 function App() {
   const boxCard = useRef({});
   const textInput = useRef({});
   const [password, setPassword] = useState([]);
   const [passwordLoaded, setPasswordLoaded] = useState(false);
   const [copied, setCopied] = useState(false);
-  const [isKata, setIsKata] = useState(false);
+  const [isKata, setIsKata] = useState(false)
```

Comment:
@ans-4175 I'm trying to test the [GitHub Action](https://github.com/ans-4175/password-ga/pull/11/files#diff-b1e4ae7d14d1dfd38f68ca8dc5fd2311ccf235690785f4b1173d1c24de7a0ebf) to verify this, but it's not running for some reasons... any ideas?

Reply:
hmm, pretty weird since others line already affected

Reply:
hmm, let me try adding `pull_request` on top of `push`: https://stackoverflow.com/questions/64802322/running-github-actions-on-the-code-in-a-pull-request-from-a-fork. Hopefully it works...

Reply:
Hmmm, no dice. Perhaps it's intended so that there won't be random contributors creating a GitHub Action PR that is malicious, for example, abusing the repository's Secrets.

Let me try something silly in my forked repo...